messages/core.json

```diff
@@ -12,7 +12,7 @@
   "OrgDataNotAvailableErrorAction2": "Use `sfdx force:config:set` to update the defaultusername.",
   "OrgDataNotAvailableErrorAction3": "Use `sfdx force:org:create` to create a new org.",
   "OrgDataNotAvailableErrorAction4": "Use `sfdx force:auth` to authenticate an existing org.",
-  "NamedOrgNotFound": "No AuthInfo found for name %s",
+  "NamedOrgNotFound": "No AuthInfo found for %s %s",
```

Comment:
Let's improve the message slightly. How about:

"No authorization information found for %s %s"